### DISCUSSION TITLE
Vertex Color support for GLTF/IQM

### DISCUSSION THREAD

**Discussion by GithubPrankster** [raysan5/raylib#1718] [Ideas]
Hello. I'm curious if vertex colors included in model formats such as `.iqm` and `.gltf/.glb` could have utilization implemented in the future! My main use for them is being able to shade/color my models, which are mainly low-poly, retro stylized.

Here's an example of a model using *solely* vertex-coloring. *(Blender 2.92 screenshot)*
![image](https://user-images.githubusercontent.com/33995085/114323664-99efd580-9afc-11eb-8901-8827936912ed.png)

The model, in *Godot Engine 3.3.rc7*. *(Utilizing custom shader to emulate PS1-like look, using `.glb` format)*
![image](https://user-images.githubusercontent.com/33995085/114324067-8fced680-9afe-11eb-9e39-94b5313355c0.png)

I believe `cgltf` supports obtaining this data through `cgltf_attribute_type_color`, and `LoadIQM` through `IQM_COLOR`.

Any thoughts?

**Comment by chriscamacho** [comment]
some time ago, I implemented multiple meshes in the wavefront OBJ loader while not per vertex (but rather per tri) it works very well for low-poly models - for example the resources at kenney.nl these low poly models are completely un textured and rely on materials having different diffuse colours.  Maybe this might be sufficient for your needs, there has recently been some issues with the OBJ loader and to be honest I'm not sure of its exact status, but you might like to give it a try ?
